src/store/provider.js

```diff
@@ -37,8 +38,9 @@ export const state = () => ({
 })
 
 export const getters = {
-  getProviderName: (state, getters, rootState) => (providerCode) => {
-    const mediaType = rootState.search.mediaType
+  getProviderName: (state) => (providerCode) => {
+    const searchStore = useSearchStore()
+    const mediaType = searchStore.mediaType
```

Comment:
`mediaType` doesn't appear to be a property of the `searchStore` :thinking: Should this be `searchType`?

Reply:
Great catch!